src/app/cardsView/base-key-view.ts

```diff
@@ -0,0 +1,27 @@
+import {CardsService} from "./services/cards-service";
+import {Card} from "../card/card";
+import {ActivatedRoute} from "@angular/router";
+/**
+ * Created by wiekonek on 22.11.16.
+ */
+
+export class BaseKeyView {
+  protected cards: Card[];
+  protected key: number;
+
+
+  constructor(protected route: ActivatedRoute, protected service: CardsService) { }
+
+  ngOnInit() {
+    console.log("Base ngOnInit().");
```

Comment:
OKej, z tego co ogarnąłem, to kolejne commity do tego brancha zostaną automatycznie dodane do requesta, więc idę wywalić te logi.